src/modules/45-dashboards/strings/strings.en.yaml

```
@@ -80,3 +80,17 @@ dashboardSortingOptions:
   recentlyCreated: 'Recently Created'
   mostViewed: 'Most Viewed'
   mostLiked: 'Most Liked'
+aida:
+  assist: I can generate new Custom Dashboard widgets quickly and effectively.
+  generating: AIDA is Generating
+  createWidget: Create a widget using AIDA
+  helpText: Harness AI Development Assistant(AIDA) can help build Custom Dashboard widgets quickly and effectively.
+  specify: Specify the {{explore}} metrics you would like to see on the visualisation
```

I think our target audience is US English, not UK
```suggestion
  specify: Specify the {{explore}} metrics you would like to see on the visualization
```